server/the-usher-openapi-spec.yaml

```diff
@@ -429,6 +429,53 @@ paths:
           $ref: '#/components/responses/NotFound'
 
   /personas:
+    get:
+      'x-swagger-router-controller': 'personas/persona'
+      operationId: getPersonas
+      summary: Retrieve a list of personas
+      parameters:
+        - name: filter
+          in: query
+          description: Filter by a field value
+          schema:
+            type: string
+            example: tenantname:value1,sub_claim:value2
```

Comment:
@k1ch  if we specify filters as colon separated values, what happens if the value has a colon in it, ie. a date field? 
Instead of using an attribute named `filter`, should each of the available search keys be their own optional query param? 
ie. key, tenantkey, sub_claim, user_context, tenantname?

Reply:
Valid point and simpler approach! I will update the parameters list and get rid of `filter`.